File: Tutorial/Python/led-button.py
#!/usr/bin/python3
import RPi.GPIO as GPIO
from time import sleep

GPIO.setmode(GPIO.BCM)
GPIO.setup(17, GPIO.OUT)
GPIO.setup(2, GPIO.IN, pull_up_down=GPIO.PUD_UP)
 
while True:
    if GPIO.input(2) == False:
        GPIO.output(17, 1)
        # GPIO.output(17, not GPIO.input(17))
        # sleep(0.3)

File: Tutorial/C/led-button.c
#include <wiringPi.h>

int main(void)
{
    wiringPiSetupGPIO();
    pinMode(17, OUTPUT);
    pinMode(2, INPUT);
    pullUpDnControl (2, PUD_UP);
    
    digitalWrite(17, 0); 
    while(1)
    {
        if(!digitalRead(2)){
            digitalWrite(17, 1);
            // digitalWrite(17, !digitalRead(17));
            delay(300);
        }
    }
    
    return 0;
}
